source/components/AppHeader.js

```diff
@@ -52,9 +57,10 @@ class AppHeader extends HTMLElement {
                         <path stroke-linecap="round" stroke-linejoin="round" d="M17.982 18.725A7.488 7.488 0 0012 15.75a7.488 7.488 0 00-5.982 2.975m11.963 0a9 9 0 10-11.963 0m11.963 0A8.966 8.966 0 0112 21a8.966 8.966 0 01-5.982-2.275M15 9.75a3 3 0 11-6 0 3 3 0 016 0z" />
                     </svg>
 
-                    <h1>Wizarding World of Fortune Telling</h1>
                 </a>
 
+                <h1 id="home-redirect">Wizarding World of Fortune Telling</h1>
```

Comment:
if you need to redirect to homepage, then you also want to delete the parent a tag